src/SourceBuild/content/repo-projects/Directory.Build.targets

```diff
@@ -642,6 +691,47 @@
     </Touch>
   </Target>
 
+  <!--
+    Add Microsoft.Build.NoTargets and Microsoft.Build.Traversal SDK overrides, for all repos that utilize them,
```

Comment:
> for all repos that utilize them

Why not just always define these overrides, regardless of it these msbuild sdks are used?

Reply:
3 different versions of `Microsoft.Build.NoTargets` and 2 versions of `Microsoft.Build.Traversal` are used by various repos. We can only override one version of each package, so we need to match the version used by specific repo. Using highest available version did not work.

Reply:
Fixed with https://github.com/dotnet/installer/pull/18478/commits/1e17ce56ede64048cd8cc068238e24ab32662045